tests/test_main_kr.py

```
@@ -986,3 +1046,485 @@ def test_use_decimal_for_multiple_of(output_file: Path) -> None:
+    golden_output="main_kr/url_with_headers/output.py",
+)
+@freeze_time("2019-07-26")
+def test_url_with_http_headers(mocker: pytest.MonkeyPatch, output_file: Path) -> None:
+    """Fetch schema from URL with custom HTTP headers.
+
+    The `--url` flag specifies a remote URL to fetch the schema from instead of
+    a local file. The `--http-headers` flag adds custom HTTP headers to the request,
+    useful for authentication (e.g., Bearer tokens) or custom API requirements.
+    Format: `HeaderName:HeaderValue`.
+    """
+    from unittest.mock import Mock  # noqa: PLC0415
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)